src/types/user.types.ts

```diff
@@ -150,7 +150,7 @@ export interface FlatCristinPerson {
     no?: string | null;
     en?: string | null;
   };
-  keywords?: Keywords[];
+  keywords?: (Keywords | KeywordsOld)[];
```

Comment:
Må ærlig si at jeg ikke henger med 100% på casting og at keywords kan være 2 typer her, da jeg ville antatt at den alltid har en type her? Tenkte kanskje at det var en modell i GET og en annen for PUT elns nå 🤔
Anyways, sjekket ut branchen og ser det ut som at visning av keywords på forskerprofil ikke virker
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/274e32f6-ec6f-4ffa-92c8-c394e5225afb)